content/docs/querying/functions.md

```diff
@@ -85,6 +95,15 @@ rate(http_requests_total{job="api-server"}[5m])
 
 `rate` should only be used with counters.
 
+## `round()`
+
+`round(v instant-vector, places=0 scalar)` rounds the sample values of all
```

Comment:
I'm not sure that 'places' is the best API to offer, as sometimes you want to round to a non-decimal base and doing round(x, log(2)/log(10)) isn't exactly the most user friendly vs round(x, 1e-2) for when you want more decimal places.

I'd suggest round(x, 5) would round to the nearest 5 etc., and have similar for floor/ceil.